### PR TITLE
Unassign Skill Points button crashes every time fix with debug_msg support

### PR DESCRIPTION
**Pull request by markm11**
https://github.com/ChaosMarc/PlugY/commit/dc1a078c35cd932116fd7958afaf00f2f1ec2b2a
https://github.com/ChaosMarc/PlugY/commit/ac64ead3a9a2f99ba10481cbb557587abe9f1fae

### CODE REVIEW

File: PlugYInstaller/PlugY.nsi [hunk 1]
```diff
@@ -203,12 +203,11 @@ Section $(SECTION_NAME_UNINSTALLER) Uninstaller
   WriteRegDWORD HKLM "Software\Microsoft\Windows\CurrentVersion\Uninstall\${NAME}" "NoRepair" 1
 SectionEnd
 
-Section $(SECTION_PATCH_FILE) PatchFile
-  SectionIn 1
-  SetOutPath $D2Path
-  File "${D2FILES}\PatchD2File.exe"
-  File "${D2FILES}\RestoreD2File.exe"
-SectionEnd
+;;;Section $(SECTION_PATCH_FILE) PatchFile
```
Comment (ChaosMarc) [inline PlugYInstaller/PlugY.nsi:206]: Why did you comment this out? The file patcher is fully working and should therefore be included

File: PlugYInstaller/PlugY.nsi [hunk 2]
```diff
@@ -224,12 +223,12 @@ Section "Uninstall" Uninstall
...
+  Delete "$D2Path\PlugY.ini"
   Delete "$D2Path\PatchD2File.exe"
   Delete "$D2Path\RestoreD2File.exe"
   Delete "$INSTDIR\PlugY.exe"
   Delete "$INSTDIR\PlugY.log"
   Delete "$INSTDIR\BnetLog.txt"
-  Delete "$INSTDIR\PlugY.ini"
   Delete "$INSTDIR\PlugY_The_Survival_Kit_-_Readme.txt"
   Delete "$INSTDIR\PlugY_The_Survival_Kit_-_LisezMoi.txt"
   Delete "$INSTDIR\PlugY_The_Survival_Kit_-_Liesmich.txt"
```
Comment (ChaosMarc) [inline PlugYInstaller/PlugY.nsi:234]: I can't comment the PlugYInstaller/PlugY_The_Survival_Kit_v12.00.exe file itself. I'm against binary files in git. IMHO this should only be included in a release

File: PlugY/D2functions.cpp [hunk 3]
```diff
@@ -18,8 +18,77 @@ s_shifting shifting;
 #define E2F(F, A, R, N, P)	T##N N;
 #define E2C(F, A, T, N)		T* pt##N;
 #define F8(X, Z, A,B,C,D,E,F,G,H,I, R, N, P) T##N N;
-#define A8(X, Z, A,B,C,D,E,F,G,H,I, R, N, P) T##N N;
-#define C8(Z, A,B,C,D,E,F,G,H,I, T, N)       T* pt##N;
+
+// Common
+TD2GetCharStatsBIN D2GetCharStatsBIN;
```
Comment (ChaosMarc) [inline PlugY/D2functions.cpp:23]: Could you explain what you did here? I actually don't get where all this code comes from 😇

**Review by ChaosMarc — CHANGES_REQUESTED** [review]

**Review by ChaosMarc — CHANGES_REQUESTED** [review]
did you already or will you update the solution to VS2019 in this PR?

**Comment by markm11** [comment]
If you could, can you just delete this pull request from me.  This is not what I wanted to do right now.   Also I sent you friend request on discord.

File: PlugYInstaller/PlugY.nsi (hunk 1 above)
Comment (markm11) [reply PlugYInstaller/PlugY.nsi:206]: When I built the installer it did not work because it could not find those files.

**Comment by markm11** [comment]
This is not the request I wanted.

**Pull request closed without merging**